static/js/materialize-tags.min.js

```diff
@@ -0,0 +1 @@
t.attr("size",Math.max(this.inputSize,$input.val().length))},self));self.$container.on("click","[data-role=remove]",$.proxy(function(event){if(self.$element.attr("disabled")){return}self.remove($(event.target).closest(".chip").data("item"))},self));if(self.options.itemValue===defaultOptions.itemValue){if(self.$element[0].tagName==="INPUT"){self.add(self.$element.val())}}},destroy:function(){var self=this;self.$container.off("keydown","input");self.$container.off("click","[role=remove]");self.$container.remove();self.$element.removeData("materialtags");self.$element.show()},focus:function(){this.$input.focus()},input:function(){return this.$input},findInputWrapper:function(){var elt=this.$input[0],container=this.$container[0];while(elt&&elt.parentNode!==container){elt=elt.parentNode}return $(elt)}};$.fn.materialtags=function(arg1,arg2,arg3){var results=[];this.each(function(){var materialtags=$(this).data("materialtags");if(!materialtags){materialtags=new TagsMaterialize(this,arg1);$(this).data("materialtags",materialtags);results.push(materialtags);$(this).val($(this).val())}else if(!arg1&&!arg2){results.push(materialtags)}else if(materialtags[arg1]!==undefined){if(materialtags[arg1].length===3&&arg3!==undefined){var retVal=materialtags[arg1](arg2,null,arg3)}else{var retVal=materialtags[arg1](arg2)}if(retVal!==undefined){results.push(retVal)}}});if(typeof arg1=="string"){return results.length>1?results:results[0]}else{return results}};$.fn.materialtags.defaults=defaultOptions;$.fn.materialtags.Constructor=TagsMaterialize;function makeOptionItemFunction(options,key){if(typeof options[key]!=="function"){var propertyName=options[key];options[key]=function(item){return item[propertyName]}}}function makeOptionFunction(options,key){if(typeof options[key]!=="function"){var value=options[key];options[key]=function(){return value}}}var htmlEncodeContainer=$("<div />");function htmlEncode(value){if(value){return htmlEncodeContainer.text(value).html()}else{return""}}function doGetCaretPosition(oField){var iCaretPos=0;if(document.selection){oField.focus();var oSel=document.selection.createRange();oSel.moveStart("character",-oField.value.length);iCaretPos=oSel.text.length}else if(oField.selectionStart||oField.selectionStart=="0"){iCaretPos=oField.selectionStart}return iCaretPos}function keyCombinationInList(keyDownEvent,lookupList){var found=false;$.each(lookupList,function(index,keyCombination){if(typeof keyCombination==="number"&&keyDownEvent.which===keyCombination){found=true;return false}if(keyDownEvent.which===keyCombination.which){var alt=!keyCombination.hasOwnProperty("altKey")||keyDownEvent.altKey===keyCombination.altKey,shift=!keyCombination.hasOwnProperty("shiftKey")||keyDownEvent.shiftKey===keyCombination.shiftKey,ctrl=!keyCombination.hasOwnProperty("ctrlKey")||keyDownEvent.ctrlKey===keyCombination.ctrlKey;if(alt&&shift&&ctrl){found=true;return false}}});return found}$(function(){$("input[data-role=materialtags]").materialtags()})})(window.jQuery);
```

Comment:
I guess these files were committed earlier but not required in static/js right ?